src/pages/Verification/Verification.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState, useEffect } from "react";
+import { Typography, Container, Box, Button } from "@mui/material";
+import { MailOutline } from '@mui/icons-material';
+import { selectToken, selectUser } from "@/redux/userSlice";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { RootState } from '@/redux/store';
+import { BASE_URL } from '@/config/default';
+import axios, { AxiosError, AxiosResponse } from "axios";
+import AlertMessage from "@/components/ui/AlertMessage";
+
+interface AuthResponse {
+  success: boolean;
+  data?: any;
+  error?: string;
+  message?: string;
+}
+
+export default function Verification() {
+  const token = useSelector((state: RootState) => selectToken(state));
```

Comment:
I think using the `(state: RootState) => selectToken(state));` is a bit redundant. The function `selectToken()` it self uses similar syntax when it was defined. All you need to do is pass the function `selectToken` without calling it and all will be good